ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CurrencyUtils, RawTransaction, RawTransactionSerde, Transaction } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { flags } from '@oclif/core/lib/parser'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class PostCommand extends IronfishCommand {
+  static summary = 'Post a raw transaction'
+
+  static description = `Use this command to post a raw transaction.
+
+  The output is a finalized posted transaction. The transaction is also added to the wallet, and sent out to the network.`
+
+  static examples = [
+    '$ ironfish wallet:post 618c098d8d008c9f78f6155947014901a019d9ec17160dc0f0d1bb1c764b29b4...',
```

Comment:
Can you add the previous step wallet:send and  next step wallet:broadcast in the examples? 

Follow what bitcoin does 
```
Create a transaction:

bitcoin-cli createrawtransaction "[{\"txid\" : \"mytxid\",\"vout\":0}]" "{\"myaddress\":0.01}"
Sign the transaction, and get back the hex:

bitcoin-cli signrawtransactionwithwallet "myhex"
Send the transaction (signed hex):

bitcoin-cli sendrawtransaction "signedhex"
```

Reply:
That's not related to this overall command, that's "how to perform a task" that's outside the function of this specific command. We can put that into the description if you like, but creating raw transactions does not exist yet, right?

Reply:
Creating a raw transaction will be `wallet:send --rawTransaction`. It does not exist yet, I will update this once I add the flag.